src/workers/webdav/worker/server.ts

```diff
@@ -19,7 +19,7 @@ export class InternxtWebdavServer {
     storageManager?: IStorageManager
   ) {
     this.server = new WebDAVServer({
-      hostname: 'localhost',
+      hostname: 'virtual.drive.internxt.com',
```

Comment:
Why is lowercase here and uppercase in the other file?

Reply:
get the IP from a host its not case sensitive (thats why here its not needed), but showing it on Finder it is

Reply:
Changed it for better consistency